src/cpp/session/modules/quarto/SessionQuarto.cpp

```diff
@@ -57,10 +57,12 @@ using namespace rstudio::core;
 
 const char * const kRStudioQuarto = "RSTUDIO_QUARTO";
 
-// ignored unused functions when quarto not enabled
-#ifndef QUARTO_ENABLED
-#pragma GCC diagnostic push
-#pragma GCC diagnostic ignored "-Wunused-function"
+#ifndef WIN32
```

Comment:
The code this is replacing checks `_WIN32` vs. `WIN32`. Just pointing that out in case this isn't an intentional change.

Reply:
You're right -- I'll fix this (and another inadvertent use I saw in this file).